packages/optlang/test_optlang.py

```diff
@@ -0,0 +1,37 @@
+def test_optlang(selenium):
+    selenium.load_package("optlang")
+    selenium.run(
+        """
+from optlang import Model, Variable, Constraint, Objective
```

Comment:
You can keep indent all this block by 8 spaces, it will still run fine and will be more readable.

Reply:
Done in the latest commit